packages/calcite-components/src/tests/utils.ts

```diff
@@ -425,3 +425,32 @@ export function toBeNumber(): any {
     },
   };
 }
+
+/**
+ *
+ * Sets the value of a CSS variable to a test value.
+ * This is useful for testing themed components.
+ *
+ * @param token - the token as a CSS variable
+ * @returns string - the new value for the token
+ */
+export function assignTestTokenThemeValues(token: string): string {
```

Comment:
Can we move this into `commonTests`? #9056 should help split up helpers by test topic.